recipes/omsl/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "omsl" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ mslversion }}
+
+source:
+  url: {{ mslsourceurl }}
+  sha256: {{ mslsourcesha256 }}
+
+build:
+  script_env:
+   - MSLVERSION={{ mslversion }}
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - omcompiler
+    - libiconv           # required by omcompiler (if not already present on the host)
+    - gcc_linux-64       # required by omcompiler (if not already present on the host)
+    - gxx_linux-64       # required by omcompiler (if not already present on the host)
+    - gfortran_linux-64  # required by omcompiler (if not already present on the host)
+    - make               # required by omcompiler (if not already present on the host)
+    - cmake              # required by omcompiler (if not already present on the host)
```

Comment:
Agreed, I will reach out to the maintainer of @conda-forge/omcompiler-feedstock asking him to add these packages in the run section.
These packages are installed on most many hosts by default. I found that they are required when installing omcompiler on a fresh container in the cloud. Can you please have a look @jschueller ?

In the first step, `omcompiler` builds C/C++ code from the mo-files (and the libraries). In the 2nd step, it calls gxx to build a binary that represents the model and returns the timeseries. The the "Modelica Standard Library" is used as a foundation for almost all Modelica models. This feedstock is intended to make it  installable through conda.

And yes, `omcompiler` expects to find the Modelica libraries either in `/lib/omlibrary/` or in `~/.openmodelica/libraries/`. The 1st is the default, e.g. when the omcompiler and the Modelica libraries are installed though `apt-get` in a Debian environment (of course, `conda` and `apt` will never be mixed). The 2nd is used by the [package manager](https://openmodelica.org/doc/OpenModelicaUsersGuide/latest/packagemanager.html) which is built in `omcompiler` enabling the installation of Modelica libraries in a mo-script via `installPackage`. Unfortunately, this cannot be used in our CI environment.

Reply:
Why aren't the mo-files precompiled and shipped and dynamically linked into end-users' models? Is this a JIT compiler-type framework where mo-files are compiled as-needed?

Reply:
feel free to send a PR against the omcompiler package

Reply:
This is indeed a JIT compiler-type framework where the mo-files are compiled as needed. In general "a model" consists of a set of (mostly) acausal "differential algebraic equations" (DAE) where the number of variables and equations is always equal. Most user models are composed from (sub-) models from the MSL (this library). The mo-files cannot be precompiled because the causality (i.e. the direction of a signal flow) depends on how a model is composed (as far as I understood). omcompiler creates the C code which has causality.